project_one/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Confidence Boost</title>
+	</head>
+	<body>
+
+    <div>
+
+    <p>
+      <h1>
+      <a  id = "grading-element-two" href="file:///Users/ramishsaqib/Documents/about.html" title="Click for a fact">About</a>
```

Comment:
no `about.html` in the repo! thus cannot find `#grading-element-two` on `about.html`